cvat/apps/quality_control/quality_reports.py

```diff
@@ -156,6 +157,7 @@ class ComparisonParameters(_Serializable):
         dm.AnnotationType.polygon,
         dm.AnnotationType.polyline,
         dm.AnnotationType.skeleton,
+        dm.AnnotationType.label,
```

Comment:
Ah thank you so much for the clarification related to `AnnotationType.label`. Another possible implementation of tag comparison can be to use a custom label matching function instead of using the one in `datumaro`. 

As you correctly mentioned to reproduce the original problem first, first looking at it then going to matching labels.

On experimenting out with few examples I didn't come across any issue in the formation of confusion matrix or any other variable of quality report. 
In that case, @Marishka17 can you please provide some more details related to the job. If possible share the backup of that task?

Reply:
Please check the answer https://github.com/opencv/cvat/issues/7424#issuecomment-1988760249